front/lib/api/workspace.ts

```diff
@@ -104,17 +104,31 @@ export async function setInternalWorkspaceSegmentation(
  * @param role RoleType optional filter on role
  * @returns UserType[] members of the workspace
  */
-export async function getMembers(
-  auth: Authenticator,
-  role?: RoleType
-): Promise<UserTypeWithWorkspaces[]> {
+export async function getMembers({
+  auth,
+  role,
+  userIds,
+}: {
+  auth: Authenticator;
+  role?: RoleType;
+  userIds?: ModelId[];
+}): Promise<UserTypeWithWorkspaces[]> {
```

Comment:
`nit` can we keep as a proper param and only bundle the optional parameters in the object?

Something like:
```suggestion
export async function getMembers(
  auth: Authenticator, {
  role,
  userIds,
}: {
  role?: RoleType;
  userIds?: ModelId[];
}): Promise<UserTypeWithWorkspaces[]> {
```

Reply:
yeah it's better